WordPress/JetpackStatsWidgets/Model/GroupedViewData.swift

```diff
@@ -1,16 +1,16 @@
-import SwiftUI
+import Foundation
```

Comment:
Is it necessary to import foundation here?

I see the change from `LocalizedString` to `String` below, but `String` is part of the standard library.

If not necessary:

```suggestion
```